src/components/TreeFilter/VirtualizedTreeView.tsx

```diff
@@ -379,14 +400,16 @@ export class VirtualizedTreeView extends React.PureComponent<IVirtualizedTreeVie
         this.setState(prevState => ({
             ...prevState,
             searchText: searchText,
-            filteredItems: newItems
+            filteredItems: newItems,

```

Comment:
what is the difference between filteredItems and searchedItemList

Reply:
Well, searchedItemList as it name says is list of searcched items. 

I spent quite some time to figure out what filtered items property is and how it works. I couldn't find why and where it set filteredItems as list of all items when we have active search field and deselect all items so I added new property that with intiutive name tell what it is.